### DISCUSSION TITLE
How to make wildcards take effect when to mount mobile phone with sshfs?

### DISCUSSION THREAD

**Discussion by hwypengsir** [termux/termux-app#3530] [Q&A] [answered]
The wildcard '*' means it will match any number of characters or a set of characters.
`test*` matched `test.rst`,it works :

    ls   /tmp/material/test*
    /tmp/material/test.rst

Prepare my mobile phone to be mounted:

    #download termux.apk and install on the mobile phone first,
    #then do the following in termux console
    pkg install openssh  
    whoami
    uo_a313
    ifconfig
    192.168.31.130

    pkg install termux-auth
    passed  #set password

    sshd  #start ssh server in the mobile phone
    termux-setup-storage

Now i can mount the mobile phone:

    sudo sshfs -p 8802 u0_a313@192.168.31.130://data/data/com.termux/files/home/storage/downloads/WeiXin    /mnt/material
    sudo ls  /mnt/material
    test.rst
    sudo ls  /mnt/material/test*
    ls: cannot access '/mnt/material/test*': No such file or directory

[![enter image description here][1]][1]

How to make `*` match any number of characters as usual when to mount mobile pjone with sshfs ?

  [1]: https://i.stack.imgur.com/zwzds.png

**Comment by agnostic-apollo** [comment, marked as answer]
https://unix.stackexchange.com/a/355681
